### PR TITLE
Link generation

### PR DESCRIPTION
**Pull request by Bhupendra-Maurya**
This pull request introduces the following key updates:

1. **Assessment Link Generation:**
   - Implemented functionality to generate a unique link for each assessment.
   - The link is generated when the "Generate Link" button is clicked for each assessment item.

2. **Search and Filtering:**
   - Added a search bar to filter assessments based on the search input.
   - Integrated search functionality for names of employees or candidates based on the selected `assessmentType` (internal or external).

3. **Select All and Individual Item Selection:**
   - Implemented a "Select All" checkbox to select or deselect all filtered items at once.
   - Added functionality to individually select or deselect employees or candidates from the list.

4. **Expiry Date:**
   - Added a date picker for selecting the expiry date, which will be used when sending the selected names.

5. **Send Functionality:**
   - Implemented a "Send" button to handle sending the selected names, expiry date, and link to the backend or for further processing.
   - The button is added, but the backend functionality is yet to be implemented.
   - For now, the data is displayed in the console.

---

image
image
image
image

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -91,7 +91,7 @@
               }
             ],
             "styles": [
-              "src/styles.css"
+              "src/styles.scss"
```
Comment (goelp30) [inline angular.json:94]: Does every one agrees to work on scss

**Comment by goelp30** [comment]
also i think we have agreed on date time picker not only date picker
image

**Comment by Bhupendra-Maurya** [comment]
> also i think we have agreed on date time picker not only date picker image



**Added date time picker**
image

**Review by sanjay611080 — APPROVED** [review]

**Review by kasharora03 — DISMISSED** [review]
logic is fine, shadow in select employee modal can be improved!

**Review by Bhupendra-Maurya — COMMENTED** [review]
image
image
image

I have reduced shadow please have a look

**Review by Harshjdev — DISMISSED** [review]

**Pull request closed without merging**